studio/styles/storage.scss

```diff
@@ -64,7 +64,7 @@ button[aria-haspopup='menu']:focus-visible {
   background: linear-gradient(
     to right,
     hsl(var(--background-surface-300)) 4%,
```

Comment:
@joshenlim maybe something like this?

<img width="566" alt="Screenshot 2023-11-14 at 11 12 07" src="https://github.com/supabase/supabase/assets/25671831/4fd2e2ad-57d8-4104-b342-fca0157558da">